docs/Public Module Imports.md

```diff
@@ -9,6 +9,7 @@ Table of Contents:
  - [Fiori Package - @ui5/webcomponents-fiori](#fiori)
  - [Icons Package - @ui5/webcomponents-icons](#icons)
  - [Icons-TNT Package - @ui5/webcomponents-icons-tnt](#icons-tnt)
+ - [Icons SAP Business Suite Package - @ui5/webcomponents-icons-business-suite](#icons-business-suite)
```

Comment:
You can also list both **tnt** and **business-suite** packages in the root Readme (in the "Structure" section).